ceph-build/build/build_deb

```diff
@@ -57,6 +57,9 @@ gen_debian_version() {
 
 bpvers=`gen_debian_version $debian_version $DIST`
 
+# look for a specific package to tell if we can avoid the build
+chacra_endpoint="ceph/${chacra_ref}/${distro}/${DIST}/${ARCH}/ceph-osd_${vers}-${bpvers}_${ARCH}.deb"
```

Comment:
using `ceph-osd` here would make this check not work for Ceph versions where the split didn't exist (Hammer?)